src/main/java/uk/gov/hmcts/reform/civil/service/PaymentRequestUpdateCallbackService.java

```diff
@@ -56,6 +58,10 @@ public void processCallback(ServiceRequestUpdateDto serviceRequestUpdateDto, Str
                 createEvent(caseData, serviceRequestUpdateDto.getCcdCaseNumber(), feeType);
             }
 
+            if (caseData.isLipvLipOneVOne() && feeType.equals(FeeType.HEARING.name())) {
```

Comment:
I am happy to approve, but I must ask if it makes sense to call the notification event directly here. All other notification events are done via camunda. Would it be better to have another event (LikeLipHearingPaymentSuccesful) that triggers a business process for this notification event?